Core/MoneyConverter.cs

```diff
@@ -12,5 +12,17 @@ public static Money Convert(this Money money, Currency currency)
             }
             throw new ArgumentException("Invalid Currency");
         }
+
+        public static Currency Convert(string currency)
+        {
+            switch (currency.ToLowerInvariant())
```

Comment:
Codacy found an issue: [Change this normalization to 'ToUpperInvariant()'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)